api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -1481,6 +1481,24 @@ private void enableSystemAdministration() {
                 9,
                 "App: coreapps.systemAdministration",
                 null)));
+
+        if (config.isComponentEnabled(Components.SYSTEM_ADMINISTRATION_INPATIENT_TOOLS)) {
+            apps.add(addToSystemAdministrationPage(app(CustomAppLoaderConstants.Apps.CLOSE_STALE_INPATIENT_ADMISSIONS,
+                    "pih.app.admin.closeStaleInpatientAdmissions.title",
+                    "fas fa-fw fa-list-ul",
+                    "pihcore/admin/staleInpatientAdmissions.page",
+                    10,
+                    "App: coreapps.systemAdministration",
+                    null)));
+
+            apps.add(addToSystemAdministrationPage(app(CustomAppLoaderConstants.Apps.CLOSE_STALE_ADMISSION_REQUESTS,
+                    "pih.app.admin.closeStaleAdmissionRequests.title",
+                    "fas fa-fw fa-list-ul",
+                    "pihcore/admin/staleAdmissionRequests.page",
+                    11,
+                    "App: coreapps.systemAdministration",
+                    null)));
+        }
```

Comment:
I'm good with moving away from the CALF, but I'm of two minds re: doing it piecemeal, but since you've started a system administration extensions json might make sense to move it there.

The reason I made it a separate component was that it's meaningless on servers that don't have inpatients (which is most), but I guess there isn't a huge downside to having it show up.